### DISCUSSION TITLE
Naomi tate games not rotating

### DISCUSSION THREAD

**Discussion by albmm74** [flyinghead/flycast#1514] [General]
Hi @flyinghead : I can't get my Naomi tate roms (Ikaruga, Psyvariar 2, Trizeal, Under Defeat) rotated. The option to rotate 90º is selected, but they still are shown in "yoko". I'm on Ubuntu 22.04 with the standalone appimage. Any suggestions? Thank you very much.

_EDIT: how can I generate a log to post it? Thanks._

**Comment by flyinghead** [comment]
It works fine here.
Flycast version?
OpenGL or Vulkan?
GPU and driver?

**Reply by albmm74** [reply]
2.2 appimage, openGL, R9 380, AMDgpu, MESA (kisak fresh updated).

How can I generate a full log? Thanks.

**Reply by flyinghead** [reply]
There won't be anything interesting in the log. You can enable it in Advanced > Log to File.

Please upgrade to Flycast 2.3

**Reply by flyinghead** [reply]
If Vulkan is supported by your GPU it is worth a try too.

**Reply by albmm74** [reply]
Tried 2.3 and Vulkan API in both versions... but my tate games do not want rotate.

[flycast.log](https://github.com/flyinghead/flycast/files/15358547/flycast.log)

**Reply by flyinghead** [reply]
The game is correctly identified:
`N[BOOT]: Game ID is [PSYVARIAR2 JAPAN]`
And rotation is enabled (which automatically selects the option in Settings):
`N[NAOMI]: EEPROM: vertical monitor orientation`

However this is surprising (and unsupported):
`N[RENDERER]: Monitor refresh rate: 30 Hz (640 x 480)`
Flycast expects a 60+ Hz monitor. Any idea why a 30 Hz refresh rate is reported?

**Reply by albmm74** [reply]
This is (I think) cause I'm ussing a 15 khz patched kernel with an arcade CRT monitor, working with a resolution of 640x480i.

XFCE desktop reports 30 hz refresh too.

I will look for a way to fix it (if possible), or I will try lo launch Flycast with switchres.

For the moment I'm running tate games with the RA core, but I really I'd like to find a solution for the standalone.

Thank you very much.
